risc0/zkvm/src/lib.rs

```diff
@@ -47,13 +47,14 @@
 //! The following feature flags are supported.
 //!
 //! Note that in order to use `risc0-zkvm` in the guest, you must disable the
-//! "prove" feature by setting `default-features = false`.
+//! default features by setting `default-features = false`.
 //!
 //! | Feature          | Target(s)         | Implies    | Description                                                                                                                                                  |
 //! | ---------------- | ----------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 //! | client           | all except rv32im | std        | Enables the client API.                                                                                                                                      |
 //! | cuda             |                   | prove, std | Enables CUDA GPU acceleration for the prover. Requires CUDA toolkit to be installed.                                                                         |
 //! | disable-dev-mode | all except rv32im |            | Disables dev mode so that proving and verifying may not be faked. Used to prevent a misplaced `RISC0_DEV_MODE` from breaking security in production systems. |
+//! | docker           | all except rv32im |            | Enables features that require Docker. In particular, this enables Groth16 proving.                                                                           |
```

Comment:
I think we should not require the docker flag to enable Groth16 proving, we should make this is a runtime issue, rather than a compile time one. Note that we should still use the docker flag for tests.

Reply:
Ah, and this is indeed how it is implemented. I've removed this line in cdcab5d62d79aec9f78a4294c8e53ccafaff988c